src/sipmpdf/functions.py

```diff
@@ -48,11 +48,12 @@ def generalized_poisson(k: np.int64,
   # assert kern.all(k >= 0)
   # assert kern.all(mean > 0)
   # assert kern.all(borel >= 0)
-
   # Use the loggamma function to ensure numerical stability
+
   return kern.exp(
     kern.log(mean) \
-    + (k - 1) * kern.log(mean + k * borel)  \
+    + (k - 1) \
+    * kern.log(mean + k * borel)  \
```

Comment:
Probably don't split this line, as we want to keep multiplicative together (split up just the additive terms